test/HasPredicateScanTest.cpp

```diff
@@ -331,11 +331,11 @@ TEST(CountAvailablePredicates, patternTrickTest) {
   CompactVectorOfStrings<Id> hasRelation(hasRelationSrc);
   CompactVectorOfStrings<Id> patterns(patternsSrc);
 
-  RuntimeInformation runtimeInfo;
+  RuntimeInformation runtimeInfo{};
```

Comment:
What difference does this make?

Reply:
There's none, but I like the explicitness